### PR TITLE
Feat theming and toggle raster

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: components/ui-radio.vue [hunk 1]
```diff
@@ -22,17 +22,16 @@
 
 .ui-radio__button-wrapper {
   display: inline-block;
-  width: calc(var(--spacing-small) * 2);
-  height: calc(var(--spacing-small) * 2);
+  width: calc(var(--spacing-default));
```
Comment (SiggyF) [inline components/ui-radio.vue]: remove calc?

File: api-mock/datasets-test.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        },
+        {
+          "dataType": "timeseries",
+          "description": "To be filled by Daniel",
+          "id": "wl",
+          "name": "Waterlevel",
+          "timeSpan": "Live",
+          "units": "m",
+          "dataServiceIds": ["H.simulated"],
+          "wmsUrl": "https://earthengine.googleapis.com/map/6e041ff999149aa6223f09403f76debb/{z}/{x}/{y}?token=6a1746b0833f2cf979233b41a5f331d0",
```
Comment (SiggyF) [inline api-mock/datasets-test.json]: broken url?

File: components/data-set-control-menu.vue [hunk 3]
```diff
@@ -14,34 +14,49 @@
...
           {{ dataset.name }}
-          <UiToggle
-            :checked="dataset.visible"
-            class="data-set-control-menu__control"
-            @change="toggleLocationDataSet(dataset.id)"
-          />
+          <div class="data-set-control-menu__control">
+            <UiToggle
+              :checked="dataset.visible"
+              class="data-set-control-menu__control"
```
Comment (SiggyF) [inline components/data-set-control-menu.vue]: why this class here?

File: components/site-navigation.vue [hunk 4]
```diff
@@ -6,20 +6,7 @@
...
-          <Icon
-            size="large"
-            name="theme-flooding"
-            fallback-name="placeholder"
-          />
-          <span class="site-navigation__text h4">Flooding</span>
-        </nuxt-link>
-      </li>
-      <li>
+      <!-- <li>
```
Comment (SiggyF) [inline components/site-navigation.vue]: remove?

File: layouts/default.vue [hunk 5]
```diff
@@ -6,39 +6,45 @@
...
-          :active-location-ids="activeLocationIds"
+        <v-mapbox-selected-point-layer
+          :geometry="geometry"
+        ></v-mapbox-selected-point-layer>
+        <v-mapbox-vector-layer
+          v-for="layer in vectorLayers"
+          :key="layer.id"
+          :layer="layer"
           :active-theme="activeTheme"
           @select-locations="selectLocations"
```
Comment (SiggyF) [inline layouts/default.vue]: `@click` `@selectFeature` ?

File: layouts/default.vue [hunk 6]
```diff
@@ -81,32 +92,28 @@ export default {
     }),
     ...mapGetters('map', [
       'activeSpatialData',
+      'allVectorData',
```
Comment (SiggyF) [inline layouts/default.vue]: activeVectorData?  vectorData?

File: store/map/index.js [hunk 7]
```diff
@@ -42,12 +44,25 @@ export const mutations = {
   clearActiveDatasetIds(state) {
     state.activeDatasetIds = []
   },
+  setActiveTheme(state, id) {
+    if (state.activeTheme.id === id) {
```
Comment (SiggyF) [inline store/map/index.js]: toggle?

File: store/map/index.js [hunk 8]
```diff
@@ -186,10 +227,17 @@ export const getters = {
       return ''
     }
   },
-  activeSpatialData({ activeLocationIds }, { activeDatasets }) {
-    const spatialLayers = activeDatasets.filter(has('metadata'))
-    const tiles = [get('spatial.tiles', head(spatialLayers))]
-    return tiles.filter(identity)
+  activeSpatialData({ datasets, activeRasterLayerId, activeDatasets }) {
+    if (activeRasterLayerId === '') return []
```
Comment (SiggyF) [inline store/map/index.js]: ''? null?

File: api-mock/datasets-test.json (hunk 2 above)
Comment (CindyvdVries) [reply api-mock/datasets-test.json]: removed all earthengine urls from mock-api datasets

File: components/data-set-control-menu.vue (hunk 3 above)
Comment (CindyvdVries) [reply components/data-set-control-menu.vue]: correct placement inside the datasetcontrolmenu

File: layouts/default.vue (hunk 6 above)
Comment (CindyvdVries) [reply layouts/default.vue]: changed to activeVectorData

File: store/map/index.js (hunk 7 above)
Comment (CindyvdVries) [reply store/map/index.js]: changed to toggleActiveTheme

**Pull request merged**